.travis.yml

```diff
@@ -1,29 +1,24 @@
+dist: focal
 language: python
 python:
-- '2.7'
-- '3.6'
-env:
-- DJANGO="django>=1.11,<2.0"
-- DJANGO="django>=2.0,<3.0"
+  - "3.7"
+  - "3.8"
+  - "3.9"
+  - "3.10"
```

Comment:
These are the currently supported Python versions according to https://www.python.org/downloads/